fava/core/file.py

```diff
@@ -110,6 +110,28 @@ def insert_entries(self, entries):
                          self.ledger.fava_options['insert-entry'])
             self.ledger.extensions.run_hook('after_insert_entry', entry)
 
+    # pylint: disable=R0201
+    def render_entries(self, entries):
+        """Return entries in Beancount format.
+
+        Only renders Balances and Transactions.
+
+        Args:
+            entries: A list of entries.
+
+        Yields:
+            The entries rendered in Beancount format.
+
+        """
+        for entry in entries:
+            if isinstance(entry, (data.Balance, data.Transaction)):
+                if isinstance(entry, data.Transaction) and entry.flag == 'S':
```

Comment:
The 'C' flag should also be excluded and you should check what other flags Beancount uses for generated entries.

Reply:
6369f70